modules/openxr/openxr_api.cpp

```diff
@@ -99,17 +99,7 @@ bool OpenXRAPI::openxr_is_enabled() {
 }
 
 OpenXRAPI *OpenXRAPI::get_singleton() {
-	if (singleton != nullptr) {
-		// already constructed, return our singleton
-		return singleton;
-	} else if (openxr_is_enabled()) {
-		// construct our singleton and return it
-		singleton = memnew(OpenXRAPI);
-		return singleton;
-	} else {
-		// not enabled, don't instantiate, return nullptr
-		return nullptr;
-	}
+	return nullptr;
```

Comment:
That seems a bit weird to always return `nullptr`?

Reply:
auch, how did I manage to do that..